### PR TITLE
[doc] Removes 'beta' from pipeline viewer doc

### PR DESCRIPTION
**Pull request by crespocarlos**
## What does this PR do?

Removes `beta` reference from pipeline viewer documentation as this feature will become GA

## Why is it important/What is the impact to the user?

This feature has been beta for a significant amount of time. We're now making this feature GA. This PR just updates the documentation removing the `beta` reference from it

## Checklist

- ~[ ] My code follows the style guidelines of this project~
- ~[ ] I have commented my code, particularly in hard-to-understand areas~
- ~[ ] I have made corresponding changes to the documentation~
- ~[ ] I have made corresponding change to the default configuration files (and/or docker env variables)~
- ~[ ] I have added tests that prove my fix is effective or that my feature works~

## Related issues

Relates [#130219](https://github.com/elastic/kibana/issues/130219)

### CODE REVIEW

**Review by matschaffer — APPROVED** [review]
LGTM - @robbavey might be able to provide a logstash-team cross check for us.

**Review by robbavey — APPROVED** [review]
LGTM, but I'll let @karenzone have final approval.

**Comment by robbavey** [comment]
@crespocarlos, @matschaffer I'm assuming this requires https://github.com/elastic/kibana/issues/125686 to be completed (in particular https://github.com/elastic/kibana/issues/130217) before this can be merged?

**Review by karenzone — APPROVED** [review]
Builds cleanly and LGTM. Thanks for the heads up!

**Comment by matschaffer** [comment]
@robbavey yep, that's the plan, we've actually moved the merge of this PR into work for https://github.com/elastic/kibana/issues/130217

**Comment by jasonrhodes** [comment]
LGTM too -- thanks everyone

**Comment by matschaffer** [comment]
Feels like we can count @karenzone and @robbavey as a @jsvd here :) Gonna click merge. 🎉

**Comment by jsvd** [comment]
Sorry for the delay here. Many many thanks from Logstash for moving this forward, I owe y'all a beverage 🍹

**Pull request merged**
